recipes/libcurl/all/test_package/conanfile.py

```diff
@@ -1,30 +1,36 @@
-from conans import ConanFile, CMake, tools
 import os
 import subprocess
 import re
 
+from conans import ConanFile, tools
+from conan.tools.cmake import CMake
+from conan.tools.layout import cmake_layout
```

Comment:
At cci, we're doing conan imports first, then system imports (without newlines between them).